cliquet/tests/support.py

```diff
@@ -71,17 +71,22 @@ class BaseWebTest(object):
     """
 
     api_prefix = "v0"
+    authorization_policy = 'cliquet.tests.support.AllowAuthorizationPolicy'
+    collection_url = '/mushrooms'
```

Comment:
I am not shocked by that, since this test case uses the `cliquet.tests.testapp`.

I would need your help because I can't think of a better way. Using constants ? What about `get_item_url()`. Ideas?